tests/test_dhcp_relay.py

```diff
@@ -16,10 +16,11 @@ def copy_ptftests_directory(ptfhost):
     ptfhost.copy(src="ptftests", dest="/root")
 
 @pytest.fixture(scope="module", autouse=True)
-def update_ptf_interface_mac_addresses(ptfhost, autouse=True):
+def ptf_configure_unique_interface_mac_addresses(ptfhost, autouse=True):
     """ Fixture which copies the change_mac.sh script to the PTF host. This fixture
         is scoped to the module, as it only needs to be performed once before
         the first test is run. The fixture also changes mac addresses of ptf containers
+        to have unique mac addresses.
     """
```

Comment:
I think the comment could be reorganized for better readability, and I think you should add the word "interfaces", like the below example:

```
    """ Fixture which copies the change_mac.sh script to the PTF host and executes
        it there, changing the MAC addresses of the PTF host's interfaces such that
        each interface has a unique MAC address.

        This fixture is scoped to the module, as it only needs to be performed once
        before the first test is run.
    """
```